Examples/UIExplorer/PushNotificationIOSExample.js

```diff
@@ -57,6 +63,11 @@ class NotificationExample extends React.Component {
           onPress={this._sendNotification}
           label="Send fake notification"
         />
+
+        <Button
+          onPress={this._sendLocalNotification}
+          label="Send fake local notification"
```

Comment:
nit: Please remove 'fake', "Send local notification" reads nicely. The notification is not fake - it is the notification sent by the example.